extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -862,15 +1014,12 @@ export class AmpAutocomplete extends AMP.BaseElement {
     }
     const activeIndex = mod(index, enabledElements.length);
     const newActiveElement = enabledElements[activeIndex];
-    this.inputElement_.value = newActiveElement.getAttribute('data-value');
 
-    // Highlight if suggest-first is present.
-    if (this.suggestFirst_) {
-      this.inputElement_.setSelectionRange(
-        this.userInput_.length,
-        this.inputElement_.value.length
-      );
-    }
+    this.binding_.displayActiveItemInInput(
+      dev().assertElement(this.inputElement_),
+      newActiveElement.getAttribute('data-value'),
```

Comment:
nit: set to an intermediate variable `const newValue = ...` just for reading clarity without signature context.